CONTRIBUTING.md

```diff
@@ -1,19 +1,29 @@
 # Contributing
 Welcome to the CONTRIBUTING.md file of "Smarticle" project. Please read this document thoroughly - follow it in all your interactions with the project.  
    
-## Pre-Requirements  
+## Dev environment setup  
 To contribute to our community project, download and install the following:  
 1.	Vagrant  
 2.	VirtualBox(or other VM that works with "Vagrant")  
 After the installations are complete successfully, by typing 'vagrant up' in the directory of the repository, the development environment will automatically be set up.  
   
 ## Pull Request Process  
 1.	Create an issue, address the Assignees and the Label.  
-2.	Clone the repository and make a contribution, commit change\s.  
-3.	Make sure the PR has been checked thoroughly before creating the new PR. todo  
+2.	Clone the repository and make a contribution, commit changes.  
+3.	Make sure the PR has been checked thoroughly before creating the new PR.
 4.	flake8 will test relevant python code.  
-5.	if the PR gets enough support from the maintainers, it will be approved and merged.  
-*	note that conflicts within the PR may cause the PR to be disapproved.  
+5.	if the PR gets enough support from the Core Reviewers(CR), it will be approved and merged.  
+	* **each PR should get at least 2 members reviews and only then the mentors will do their code reviews.**
+	* **conflicts within the PR may cause the PR to be disapproved.**  
```

Comment:
I think you can remove this one also, it is covered in next section

Reply:
done